tests/Oryx.BuildImage.Tests/PythonSampleAppsTest.cs

```diff
@@ -494,8 +494,9 @@ public void GeneratesScript_AndBuilds_UsingCustomVirtualEnvironmentName()
             var appDir = volume.ContainerDir;
             var appOutputDir = "/tmp/app-output";
             var script = new ShellScriptBuilder()
-                .AddBuildCommand($"{appDir} -o {appOutputDir} -p virtualenv_name={virtualEnvironmentName}")
-                .AddDirectoryExistsCheck($"{appOutputDir}/{virtualEnvironmentName}")
+                .AddBuildCommand($"{appDir} -i /tmp/int -o {appOutputDir} -p virtualenv_name={virtualEnvironmentName}")
```

Comment:
here virtualenv_name

Reply:
That is the name of the 'property' for build command and in case of generating the startup script we are using the switch called `-virtualenvname`...yeah, probably both could have been the same, but changing it now would break Jenny though.